ext/fast_polylines/fast_polylines.c

```diff
@@ -24,6 +24,18 @@
 #define rdbg(...)
 #endif
 
+typedef unsigned int uint;
+
+// Speed up the code a bit by not calling pow(10.0, precision) since it's always 10^precision
+// Convert that into a lookup instead of doing the calculation
+const double PRECISION_VALUES[MAX_PRECISION+2] = {1.0, 10.0, 1.0e+2, 1.0e+3, 1.0e+4, 1.0e+5, 1.0e+6, 1.0e+7, 1.0e+8, 1.0e+9, 1.0e+10, 1.0e+11, 1.0e+12, 1.0e+13, 1.0e+14};
```

Comment:
shouldn't this one be enough ?
```suggestion
const double PRECISION_VALUES[MAX_PRECISION+1] = {
  1.0, 10.0, 1.0e+2, 1.0e+3, 1.0e+4, 1.0e+5, 1.0e+6, 1.0e+7,
  1.0e+8, 1.0e+9, 1.0e+10, 1.0e+11, 1.0e+12, 1.0e+13
 };
```

And I've also splitted lines in two to make it more readable (one of our usual code conventions at klaxit is 80 chars per line)

Reply:
1. Yes, you're right. It would work at +1 - I remember there was something I was thinking about when I did that. But I checked my notes again, and yes, +1 is fine (also because there is a check before this is looked up)

2. By all means, splitting at 80 is sensible.